AUDIO/PlayPain.md

```diff
@@ -5,27 +5,52 @@ ns: AUDIO
 
 ```c
 // 0xBC9AE166038A5CEC 0x874BD6CB
-void PLAY_PAIN(Ped ped, int painID, int p1);
+void PLAY_PAIN(Ped ped, int painID, float p3, bool createNetEvent);
 ```
 
-```
-Needs another parameter [int p2]. The signature is PED::PLAY_PAIN(Ped ped, int painID, int p1, int p2);  
-Last 2 parameters always seem to be 0.  
-EX: Function.Call(Hash.PLAY_PAIN, TestPed, 6, 0, 0);  
-Known Pain IDs  
-________________________  
-1 - Doesn't seem to do anything. Does NOT crash the game like previously said. (Latest patch)  
-6 - Scream (Short)  
-7 - Scared Scream (Kinda Long)  
-8 - On Fire  
-```
+## Description
```

Comment:
The description shouldn't start with a `## Description` tag, anything in between the top most codeblock and the `## Parameters` section will be considered the description by default.

Reply:
Got rid of the description tag, for readability purposes (when people go through the native docs) I switched over to bold text style titles.